extra-python/pypy3/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Replacing openssl-1.1.1n source's url in build_cffi_imports.py..."
+sed -i 's/source\/openssl-1.1.1n/source\/old\/1.1.1\/openssl-1.1.1n/' "$SRCDIR"/lib_pypy/pypy_tools/build_cffi_imports.py
```

Comment:
I recommend wrapping this line with `\`. Not a demand, just something to think about - I try to make my script < 80 columns wide, if possible.